tool/test-bundled-gems.rb

```diff
@@ -33,6 +34,14 @@
     test_command << " stdlib_test validate"
 
     first_timeout *= 3
+
+    # copied from debug gem
+    build_dir = 'ext/-test-/gems/rbs'
+    FileUtils.mkdir_p(build_dir)
+    extconf_path = File.expand_path('../../gems/src/rbs/ext/rbs_extension/extconf.rb', __FILE__)
+    system("#{ruby} -C #{build_dir} #{extconf_path}") or raise
+    system("cd #{build_dir} && make extout=../../../../.ext libdir=../../../..") or raise
```

Comment:
```suggestion
    system("#{ruby} -C #{build_dir} #{extconf_path}", exception: true)
    system("make", "extout=../../../../.ext", "libdir=../../../..", chdir: build_dir, exception: true)
```

Reply:
The part is copied from the process of debug gem. Let's fix it later!